config/samples/kuadrant_v1alpha1_dnspolicy.yaml

```
@@ -0,0 +1,13 @@
+---
+apiVersion: kuadrant.io/v1alpha1
+kind: DNSPolicy
+metadata:
+  name: dnspolicy-sample
+spec:
+  targetRef:
+    name: example-gateway
+    group: ""
```

I see the [original sample](https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/config/samples/dnspolicy.yaml) didn't have a group set, is there a reason why this isn't the same as the tlspolicy sample:
```suggestion
    group: gateway.networking.k8s.io
```

---

No reason, added it now.